<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela</class>
 <widget class="QMainWindow" name="Janela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>709</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>688</width>
    <height>709</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>688</width>
    <height>709</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NMEA Packet decoder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLineEdit" name="satelites_ativos">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="satelites_visiveis">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="satativos_l">
      <property name="text">
       <string>Satelites ativos: </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="satvisiveis_l">
      <property name="text">
       <string>Satelites vísiveis:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="com"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QGroupBox" name="sats_gp">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Satelites (Visiveis e Ativos)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="satelite_layout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="0" rowspan="3">
     <widget class="QGroupBox" name="gps_gp">
      <property name="title">
       <string>GPS</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="hora_l">
         <property name="text">
          <string>Hora (UTC)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hora">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="latitude_l">
         <property name="text">
          <string>Latitude:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="latitude">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="longitude_l">
         <property name="text">
          <string>Longitude:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="longitude">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="precisao_l">
         <property name="text">
          <string>Diluição da precisão horizontal:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="precisao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="altitude_l">
         <property name="text">
          <string>Altitude (m):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="altitude">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="velocidade_l">
         <property name="text">
          <string>Velocidade (Km/h)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="velocidade">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mapa_bot">
         <property name="text">
          <string>Mapa</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>precisao</zorder>
      <zorder>hora</zorder>
      <zorder>altitude</zorder>
      <zorder>longitude</zorder>
      <zorder>latitude</zorder>
      <zorder>velocidade</zorder>
      <zorder>mapa_bot</zorder>
      <zorder>hora_l</zorder>
      <zorder>longitude_l</zorder>
      <zorder>precisao_l</zorder>
      <zorder>velocidade_l</zorder>
      <zorder>altitude_l</zorder>
      <zorder>latitude_l</zorder>
      <zorder>verticalSpacer</zorder>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="abrir_bot">
        <property name="text">
         <string>Abrir COM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fechar_bot">
        <property name="text">
         <string>Fechar COM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_bot">
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1" rowspan="3">
     <widget class="QGroupBox" name="satelite_gp">
      <property name="title">
       <string>Satelite</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="svprn_l">
         <property name="text">
          <string>SV PRN Nº</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="svprn">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="elevacao_l">
         <property name="text">
          <string>Elevação em graus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="elevacao">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="azimute_l">
         <property name="text">
          <string>Azimute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="azimuth">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sinal_l">
         <property name="text">
          <string>Sinal (db)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sinal">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="satview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>347</width>
           <height>216</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIdioma">
    <property name="title">
     <string>&amp;Idioma</string>
    </property>
    <addaction name="actionIngles"/>
    <addaction name="separator"/>
    <addaction name="actionPortugu_s"/>
   </widget>
   <addaction name="menuIdioma"/>
  </widget>
  <action name="actionIngles">
   <property name="text">
    <string>&amp;Inglês</string>
   </property>
  </action>
  <action name="actionPortugu_s">
   <property name="text">
    <string>&amp;Português</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
